app/javascript/application.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './component/App';
+
```

Comment:
- [ ] Kindly address this path. It is supposed to be `./components/App` and not `./component/App` as it i stopping your components from being rendered.